src/icons/ChevronDown/ChevronDown.native.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path } from 'react-native-svg';
+import { getThemeColors } from '../../_helpers/theme';
+
+export default function ChevronDown({ width, height, fill }) {
+  return (
+    <Svg
+      width={width}
+      height={height}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <Path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+        fill={fill}
+      />
+    </Svg>
+  );
+}
+
+ChevronDown.propTypes = {
+  width: PropTypes.number,
+  height: PropTypes.number,
+  fill: PropTypes.oneOf(getThemeColors()),
+};
+
+ChevronDown.defaultProps = {
+  width: 24,
+  height: 24,
+  fill: 'rgba(43, 131, 234, 1.0 )',
```

Comment:
```suggestion
  fill: 'rgba(43, 131, 234, 1.0)',
```

Reply:
Also, color codes shouldn't be hardcoded like this in components.

Reply:
How about `fill: colors.sapphire[800]`, where colors is imported from `"/tokens/colors"`

Reply:
Yes, that would make more sense.